azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_sonarcloud.go

```diff
@@ -32,9 +32,9 @@ func ResourceServiceEndpointSonarCloud() *schema.Resource {
 func expandServiceEndpointSonarCloud(d *schema.ResourceData) (*serviceendpoint.ServiceEndpoint, *uuid.UUID, error) {
 	serviceEndpoint, projectID := doBaseExpansion(d)
 	serviceEndpoint.Authorization = &serviceendpoint.EndpointAuthorization{
-		Scheme: converter.String("UsernamePassword"),
+		Scheme: converter.String("Token"),
 		Parameters: &map[string]string{
-			"username": d.Get("token").(string),
+			"token": d.Get("token").(string),
```

Comment:
This should be 
```go
			"apitoken": d.Get("token").(string),
```
![image](https://user-images.githubusercontent.com/57888764/199169241-3202a2d0-5f3d-430a-90cd-758b8ead162b.png)

Reply:
Thank you, I have changed it.